### PR TITLE
Update tooling to be compatible with Openshift 4.8

### PR DESCRIPTION
**Pull request by JPinkney**
Once OpenShift 4.8 releases i'll add in the new oc version and then make the PR ready to review

Signed-off-by: Josh Pinkney <joshpinkney@gmail.com>

### CODE REVIEW

File: build/template.Dockerfile [hunk 1]
```diff
@@ -34,6 +34,8 @@ RUN \
     # install rhoas
     ln -s /opt/rhoas/bin/rhoas /usr/local/bin/rhoas && \
     rhoas completion bash > $COMPDIR/rhoas && \
+    # install submariner
+    ln -s /opt/submariner/subctl /usr/local/bin/subctl && \
```
Comment (sleshchenko) [inline build/template.Dockerfile:38]: does it have autocompletion for bash?

File: README.md [hunk 2]
```diff
@@ -4,14 +4,15 @@ Default OpenShift Console Web Terminal tooling container.
 
 Includes tools that a Kubernetes and OpenShift developer would like find in their terminal:
 - [jq](https://github.com/stedolan/jq)
-- [oc](https://github.com/openshift/origin) [4.7.0](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.7.0)
-- [kubectl](https://github.com/kubernetes/kubectl) [v1.20.1](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.7.0)
-- [odo](https://github.com/openshift/odo) [v2.0.4](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/odo/v2.0.4)
+- [oc](https://github.com/openshift/origin) [4.8.2](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.8.2)
+- [kubectl](https://github.com/kubernetes/kubectl) [v0.21.0-beta.1](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.8.2)
```
Comment (sleshchenko) [inline README.md:8]: Well, is it right that beta is here?

**Review by sleshchenko — APPROVED** [review]

File: build/template.Dockerfile (hunk 1 above)
Comment (JPinkney) [reply build/template.Dockerfile:38]: Not as far as I can see

File: README.md (hunk 2 above)
Comment (JPinkney) [reply README.md:8]: Yeah, i'm not sure why they based it off the beta version. I'm going to wait for 4.8.3 and maybe they might be using a more stable version of kubectl

Comment (JPinkney) [reply README.md:8]: Turns out they didn't create a new oc version so I guess the OpenShift release just uses a beta version of kubectl :shrug:

**Pull request merged**